docker-entrypoint.sh

```diff
@@ -273,6 +273,19 @@ if expr "$1" : "apache" 1>/dev/null || [ "$1" = "php-fpm" ] || [ "${NEXTCLOUD_UP
         fi
     ) 9> /var/www/html/nextcloud-init-sync.lock
 
+    # check if config files of user installation differ from image
+    for cfgPath in /usr/src/nextcloud/config/*.php; do
+        cfgFile=$(basename "$cfgPath")
+
+        if [ "$cfgFile" != "config.sample.php" ]; then
+            if ! cmp -s "/usr/src/nextcloud/config/$cfgFile" "/var/www/html/config/$cfgFile"; then
+                echo "Warning: your config folder is missing the file $cfgFile that comes with this Docker image or it differs from the original file."
+                echo "This could cause certain Docker configuration environment variables to be ignored or not work as expected."
+                echo "Please copy /usr/src/nextcloud/config/$cfgFile into /var/www/html/config/ to ensure that all Docker configuration environment variables work as intended."
```

Comment:
Maybe: 

```suggestion
                echo "This can happen if you've previously upgraded since configuration files stored on persistent volumes are not overwritten automatically."
                echo "To fix: copy /usr/src/nextcloud/config/$cfgFile into /var/www/html/config/ to ensure that all Docker configuration environment variables work as intended."
```